### ISSUE TITLE
GetCryptoProviderForSha256 gives NullReferenceException when running locally

### ISSUE THREAD

**Issue by amervitz** [Azure/Azure-Functions#1257]
Hello,

I am having the same issue described at https://github.com/Azure/Azure-Functions/issues/683 where `Microsoft.IdentityModel.Clients.ActiveDirectory.CryptographyHelper.GetCryptoProviderForSha256(RSACryptoServiceProvider rsaProvider)` throws a null reference exception when running a v1 runtime project launched from within Visual Studio.

The issue said to open a new issue to obtain information on how to fix this

The issue links to [this commit](https://github.com/Azure/azure-functions-host/commit/7addbeeb687a43a25acb2535c180b0f3ea512964) to change the httpRuntime targetFrameworkVersion but I don't know how to do this on function that is run locally.

Thanks.

**Comment by ColbyTresness** [comment]
@soninaren

**Comment by aschuhardt** [comment]
@amervitz Did you ever find a solution for this issue?  Currently blocked by it myself.

**Comment by amervitz** [comment]
@aschuhardt Unfortunately I did not find a solution for this specific error. It's hard to remember due to the passage of time but I think we came up with a workaround for local development, possibly using a hacky assembly version redirection technique or had a seperate code path used just for local development.

**Comment by aschuhardt** [comment]
@amervitz Thanks for responding.  Disappointing to hear.
